scipy/conftest.py

```diff
@@ -216,6 +217,9 @@ def xp(request):
 
     Please read: https://docs.scipy.org/doc/scipy/dev/api-dev/array_api.html
```

Comment:
Optional: Have to admit this sentence is not _that_ helpful. Consider linking to a specific section of this document?